b2sdk/sync/policy.py

```diff
@@ -29,7 +29,29 @@ class AbstractFileSyncPolicy(object):
     DESTINATION_PREFIX = NotImplemented
     SOURCE_PREFIX = NotImplemented
 
-    def __init__(self, source_file, source_folder, dest_file, dest_folder, now_millis, args):
+    NEWER_FILE_SKIP = 101
+    NEWER_FILE_REPLACE = 102
+
+    NEWER_FILE_MODES = [NEWER_FILE_REPLACE, NEWER_FILE_SKIP]
```

Comment:
Is there no mode for "raise an error if dest file is newer"?

Reply:
Oh.  Is `None` the other choice?

Reply:
looks like it. I'll fix it.